cv.py

```diff
@@ -156,20 +152,37 @@ def run_cv(self):
 
         return angle, distance
 
-# holds rectangle properties
-class Rectangle:
-
-    def __init__(self, x, y, width, height, area):
-        self.x = x
-        self.y = y
-        self.width = width
-        self.height = height
-        self.area = area
 
+# this class defines a rotated rectantle around a countour
 class RotatedRectangle:
 
     # box is a 2d list of rectangle coordiantes
     def __init__(self, box, area, rot_angle):
         self.box = box
         self.area = area
         self.rot_angle = rot_angle
+
+        points = []
+        for coordinates in box:
+            points.append(Point(coordinates))
+
+        #sorts points based on y value
+        points.sort(key= lambda x: x.y)
+        #lowest point
```

Comment:
This feels really redundant when you already save box as an array. Why don't we just sort box, or make the points array a property of RotatedRectangle? This runs the risk of the values in box going out of sync with the individual points.

Reply:
When we tried to sort box itself, I believe there was some error that I didn't know how to fix. When I did it this way, it worked without any issues. I can try to change it to sort box itself.

Reply:
My understanding is that box is an array of tuples, so there's an extra layer of indexing for your lambda function. Do you ever use box after converting to points? If not it might make more sense to save self.points instead of keeping self.box around.

Reply:
BTW feel free to message me on Slack about random Python stuff, I kinda do this for a living :P

Reply:
Yeah we don't ever use box again so we can just store self.points as an array of Point objects. I thought it would be easier to save each individual point so that for readability when we use those points in run_cv.

Reply:
Sure, in that case maybe we can give them better names. point_topleft? .tl/.tr/.bl/.br? Lots of good options available, as long as it's consistent and commented.

Reply:
We felt that points1- point4 is the best option because it measures from y value, point 1 being the lowest point and point 4 is the highest point. Since the targets are at angles and the boxes are rotated it won't always be top left, top right, bottom left, or bottom right.